test/GlobalHotKeys/DeleteAndBackSpaceAliases.spec.js

```diff
@@ -1,50 +1,83 @@
 import React from 'react';
-import {mount} from 'enzyme';
-import {expect} from 'chai';
+import { mount } from 'enzyme';
+import { expect } from 'chai';
 import sinon from 'sinon';
 import simulant from 'simulant';
 
 import { GlobalHotKeys } from '../../src/';
 
-const keyAliases = ['del', 'backspace', 'Delete', 'BackSpace' ];
-const keyNames = [ 'Delete', 'Backspace' ];
+['backspace', 'BackSpace'].forEach(function(keyAlias) {
+  describe(`When an action is defined using the combination '${keyAlias}':`, () => {
+    context(`and a Backspace event is triggered`, () => {
```

Comment:
⬜️ Remove the use of backticks for this string - it no longer needs interpolation.